.vib/thanos/queryfrontend/cypress/cypress/integration/thanos_queryfrontend_spec.js

```diff
@@ -0,0 +1,43 @@
+/// <reference types="cypress" />
+
+it('allows to see runtime & build information', () => {
+  cy.visit('/');
+  cy.contains('.dropdown-toggle', 'Status').click();
+  cy.contains('[class="dropdown-item"]', 'Runtime & Build Information').click();
+  cy.contains('Runtime Information').should('be.visible');
+  cy.get('h2').contains('Build Information').should('be.visible');
+})
+
+it('allows the execution of a query', () => {
+  cy.visit('/graph');
+  cy.get('.alert').contains('No data queried yet').should('be.visible');
```

Comment:
```suggestion
  cy.contains('.alert', 'No data queried yet');
```
Same comment about chaining and `be.visible`